iohub/clearcontrol.py

```diff
@@ -425,10 +425,20 @@ def metadata(self) -> Dict[str, Any]:
 
         return metadata
 
+    @property
+    def scale(self) -> Tuple[float, ...]:
```

Comment:
`Tuple` is [deprecated](https://docs.python.org/3/library/typing.html#typing.Tuple) for 3.9+.

We still have legacy typing stuff lying around but I think new code should use generics.

Reply:
Also is there a reason to use ellipsis if the length is fixed?

Reply:
You're right, I removed the `Tuple` usage and converted it to a fixed-length tuple. Thanks!